wordpress-mobile/WordPress-iOS
# Issue/6563 strip html

Fixes #6563 

To test: 
Create a blog post with html in title and body
Publish
View posts in blog posts
Both the title and snippet should have the html tags stripped, just like in Calypso


Needs review: @nheagy can you review this for me too?
